hardhat.config.ts

```diff
@@ -4,7 +4,15 @@ require("dotenv").config();
 import './tasks'
 
 const config: HardhatUserConfig = {
-  solidity: "0.8.9",
+  solidity: {
+    version: "0.8.9",
```

Comment:
Open Question: any reason we picked this version of Solidity?

I typically use `0.8.15`.